uptane-standard.md

```diff
@@ -748,7 +748,10 @@ If the time server is implemented, the primary SHALL use the following procedure
 
 1. Gather the tokens/nonces from each secondary ECU's version report ({{version_report}}).
 2. Send the list of tokens to the time server to fetch the current time. The time server responds as described in {{time_server}}, providing a cryptographic attestation of the last known time.
-3. Verify that the signatures are valid, and that the time the server attests is greater than the previous attested time.
+3. If the time server's response meets the criteria below, update the device's clock and retain the time server's response for distribution to Secondaries, otherwise discard it and proceed without an updated time.  The criteria for checking the time server's response are:
+  - The signature over the time server's response is valid.
+  - The tokens/nonces provided to the time server have been included in the response.
+  - The time in the time server's response is later than the last time verified in this manner.
```

Comment:
A few syntactical issues:
1. Instead of "Secondaries", we've normally been saying "secondary ECUs". The "ECU" might be unnecessary, but in any case, I think "secondaries" should not be capitalized.
2. Can we just use "tokens" or "nonces"? It's weird to use both. (Not unique to these changes.)
3. Instead of referring to the "device", it might make more sense to refer to the "primary", as in "the primary's clock".

Reply:
Fixed accordingly, thanks. :)